src/app/pages/wallets/management/wallet-editor/pages-wallets-management-editor.service.ts

```diff
@@ -23,6 +25,9 @@ export class PagesWalletsManagementEditorService {
       {
         data: { name: wallet?.name ?? '' },
       }).afterClosed().pipe(
+      tap(() => {
+        wallet ? this.loadingSnackbarService.show('Updating wallet') : this.loadingSnackbarService.show('Saving wallet');
```

Comment:
Because both updating wallet and creating wallet is basically "saving" wallet we can probably just show text "Saving wallet". I will adjust issue :)